pkg/controller/jenkins/configuration/base/resources/base_configuration_configmap.go

```diff
@@ -9,6 +9,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	BASIC_SETTING_GROOVY                   = "1-basic-settings.groovy"
+	ENABLE_CSRF_GROOVY                     = "2-enable-csrf.groovy"
+	DISABLE_USAGE_STATS_GROOVY             = "3-disable-usage-stats.groovy"
+	ENABLE_MASTER_ACCESS_CONTROL_GROOVY    = "4-enable-master-access-control.groovy"
+	DISABLE_INSECURE_FEATURES_GROOVY       = "5-disable-insecure-features.groovy"
+	CONFIGURE_KUBERNETES_PLUGIN_GROOVY     = "6-configure-kubernetes-plugin.groovy"
+	CONFIGURE_VIEWS_GROOVY                 = "7-configure-views.groovy"
+	DISABLE_JOB_DSL_SCRIPT_APPROVAL_GROOVY = "8-disable-job-dsl-script-approval.groovy"
+)
```

Comment:
The golang convention for the constants is different `BASIC_SETTING_GROOVY` -> `basicSettingsGroovyScriptName`.

Reply:
changed